model/riscv_insts_aext.sail

```diff
@@ -131,27 +131,27 @@ function clause execute (STORECON(aq, rl, rs2, rs1, width, rd)) = {
     match ext_data_get_addr(rs1, zeros(), Write(Data), width_bytes) {
       Ext_DataAddr_Error(e)  => { ext_handle_data_check_error(e); RETIRE_FAIL },
       Ext_DataAddr_OK(vaddr) => {
-        if not(is_aligned(vaddr, width))
-        then { handle_mem_exception(vaddr, E_SAMO_Addr_Align()); RETIRE_FAIL }
+        if not(is_aligned(virtaddr_bits(vaddr), width))
```

Comment:
I would probably turn `is_aligned` into an overload that can take bits(), physaddr or virtaddr. Maybe not in this PR...